modules/nw-ingress-gateway-api-manage-succession.adoc

```diff
@@ -0,0 +1,59 @@
+// Modules included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/ingress-gateway-api.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-gateway-api-manage-succession_{context}"]
+= Preparing for Gateway API management succession by the Ingress Operator
+
+Starting In {product-title} 4.19, the Ingress Operator manages the lifecycle of any Gateway API custom resource definitions (CRDs). Upgrading from a version before 4.19 of {product-title} where this management was not present requires you to replace or remove any Gateway API CRDs that already exist in the cluster so that they conform to the specific {product-title} specification required by the Ingress Operator. {product-title} version 4.19 requires Gateway API Standard version 1.2.0 CRDs.
+
+[WARNING]
+====
+Updating or deleting Gateway API resources can result in downtime and loss of service or data. Be sure you understand how this will affect your cluster before performing the steps in this procedure. If necessary, back up any Gateway API objects in YAML format in order to restore it later.
+====
+
+.Prerequisites
+. Install the OpenShift CLI (`oc`).
+. You have access to an {product-title} account with cluster administrator access.
+. Optional: You have backed up any necessary Gateway API objects.
+
+.Procedure
+
+. Get a list of all the Gateway API CRDs that you need to remove by running the following command:
++
+[source,terminal]
+----
+$ oc get crd | grep "gateway.networking"
+----
++
+.Example output
++
+[source,text]
+----
+gatewayclasses.gateway.networking.k8s.io
+gateways.gateway.networking.k8s.io
+grpcroutes.gateway.networking.k8s.io
+httproutes.gateway.networking.k8s.io
+referencegrants.gateway.networking.k8s.io
+----
+
+. Delete the Gateway API CRDs from the previous step by running the following command:
++
+[source,terminal]
+----
+$ oc delete crd gatewayclasses.networking.k8s.io && \
+oc delete crd gateways.networking.k8s.io && \
+oc delete crd grpcroutes.gateway.networking.k8s.io && \
+oc delete crd httproutes.gateway.networking.k8s.io && \
+oc delete crd referencesgrants.gateway.networking.k8s.io
+----
+
+Any controller that was previously managing the lifecycle of the Gateway API CRDs will fail to operate properly. Attempting to force its use in conjunction with the Ingress Operator to manage Gateway API CRDs might prevent the cluster upgrade from succeeding.
+
+. Get the supported Gateway API CRDs using the following command:
```

Comment:
Very soft suggestion to rephrase the wording to be more parallel with the other steps:
```suggestion
. Get the supported Gateway API CRDs running the following command:
```